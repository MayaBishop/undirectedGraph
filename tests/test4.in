//test all failures
i City0
i City0
setd City0;City1;10.0
i City1
setd City0;City1;-10.0
setd City0;City1;10.0
i City2
s City3
s City1
degree City4
degree City0
d City0;City3
d City0;City2
d City0;City1
shortest_d City0;City4
shortest_d City0;City1
shortest_d City0;City0
shortest_d City0;City2
print_path City0;City4
print_path City0;City1
print_path City0;City0
print_path City0;City2
