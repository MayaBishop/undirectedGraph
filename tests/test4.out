success
failure
failure
success
failure
success
success
not found
found City1
failure
degree of City0 1
failure
failure
direct distance City0 to City1 10
failure
shortest distance City0 to City1 10
failure
failure
failure
City0 City1
failure
failure
